docs/README.md

```diff
@@ -6,7 +6,7 @@ Building Docs
 
 Look in the `_build/html` directory for output. You can also run
 
-    python3 -m SimpleHTTPServer
+    python3 -m http.server
```

Comment:
`SimpleHTTPServer` changes to `http.server` moving to python3.  I relearned that while working on the docs.